ci-operator/config/red-hat-storage/mcg-osd-deployer/red-hat-storage-mcg-osd-deployer-main.yaml

```diff
@@ -0,0 +1,57 @@
+binary_build_commands: make build
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.17
+promotion:
+  name: "4.9"
+  namespace: ocp
+resources:
+  '*':
+    limits:
+      memory: 4Gi
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tag_specification:
+  name: "4.9"
+  namespace: ocp
+tests:
+- as: test-suite
+  steps:
+    cluster_profile: aws
+    test:
+    - as: test-suite-aws
+      commands: make test
+      from: src
+      resources:
+        requests:
+          cpu: 100m
+          memory: 200Mi
+- as: test-coverage
+  steps:
+    cluster_profile: aws
+    test:
+    - as: test-coverage-aws
+      commands: make coverage
+      from: src
+      resources:
+        requests:
+          cpu: 100m
+          memory: 200Mi
+- as: lint
+  steps:
+    cluster_profile: aws
+    test:
+    - as: lint-aws
+      commands: make lint
+      from: src
+      resources:
+        requests:
+          cpu: 100m
+          memory: 200Mi
```

Comment:
Are these right?  Maybe you need a test cluster for your `make test`, but I highly doubt you need one for coverage or lint tests.

Reply:
Makes sense. Do you mean I should use `USE_EXISTING_CLUSTER=1` for both of these commands then?

Reply:
I don't know how your tests are structured, but `cluster_profile` is (part of) what is used to request that a temporary cluster be created for every execution of your test (see https://docs.ci.openshift.org/docs/architecture/step-registry/#ci-operator-test-configuration).

You certainly don't need one just to execute a `lint` program over your source code.  As they're written, your tests do not fully request one and instead just take one spot from the shared pool of cluster unnecessarily.

Reply:
Using existing cluster to run `lint` step now, and merged `test-coverage` into `test-suite`, so just two steps now: https://github.com/red-hat-storage/mcg-osd-deployer/pull/30

@bbguimaraes Is this okay?